dbschema/seeds/017.partnerships.edgeql

```diff
@@ -0,0 +1,65 @@
+with
+partnershipsJson := to_json('[
```

Comment:
FWIW I prefer the `$$` for these huge json strings. Easier to not mess up escapes.

Reply:
Will change back. I just updated to be similar to what everyone else had, but I like $$ too